packages/flutter/test/material/progress_indicator_test.dart

```diff
@@ -772,13 +774,14 @@ void main() {
       ),
     ), const Duration(seconds: 2));
 
-    tester.binding.setSurfaceSize(animationSheet.sheetSize());
-
-    final Widget display = await animationSheet.display();
-    await tester.pumpWidget(display);
-
+    final ui.Image image = await animationSheet.collate(20);
+    // The deprecated `sheetSize` is used only for the migration from
```

Comment:
What does this comment mean? Should these be migrated to the new API?

Reply:
I plan to submit a new PR for the migration because the migration will break golden tests (because they'll be 1/3 in size), and in this way we're more sure what exactly caused the golden tests. It does look a bit redundant. Should I do them in the same PR?